### PR TITLE
Price will now be saved with decimal (10, 2)

### LINKED ISSUE #191: Listing price doesn't allow decimals
**Describe the bug**
Listing price doesn't allow decimals so 950.00 or 950.99 wouldn't work

**To Reproduce**
1. Click to create a new listing
2. Fill in all the required fields
3. Enter a price of 999.99

**Expected behavior**
Should be able to support decimals

### PR DESCRIPTION
**Pull request by gilhrpenner**
This changes the current db schema from bigInt to decimal (10, 2) for listing prices

Closes #191

### CODE REVIEW

File: backend/src/repositories/prisma/PrismaListingRepository.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ class PrismaListingRepository implements IListingRepository {
             },
             data: {
                 title: listing.title,
-                price: BigInt(listing.price),
+                price: listing.price,
```
Comment (MarkRobitaille) [inline backend/src/repositories/prisma/PrismaListingRepository.ts:45]: Should this be `new Prisma.Decimal(listing.price)` to match the create method?

Comment (gilhrpenner) [reply backend/src/repositories/prisma/PrismaListingRepository.ts:45]: I don't think it is necessary, I have made a few tests and it works just fine as this is only fetching the data which is already in decimal.

Comment (MarkRobitaille) [reply backend/src/repositories/prisma/PrismaListingRepository.ts:45]: Should we remove `new Prisma.Decimal(listing.price)` from create method and instead use `listing.price` then just to be consistent?

Comment (gilhrpenner) [reply backend/src/repositories/prisma/PrismaListingRepository.ts:45]: Fixed :)

**Pull request merged**